### PR TITLE
Renaming widget component, adding admin component

### PR DESCRIPTION
**Pull request by quinnbudan**

### CODE REVIEW

File: d2l-custom-teacher-course-creation-admin.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	}
+
+	render() {
+		return html`
+			<h2>Hello ${this.prop1}!</h2>
+			<div>Localization Example: ${this.localize('myLangTerm')}</div>
+		`;
+	}
+}
+customElements.define('d2l-custom-teacher-course-creation', CustomTeacherCourseCreation);
```
Comment (BraydonKains) [inline d2l-custom-teacher-course-creation-admin.js:53]:
```suggestion
customElements.define('d2l-course-creation-admin', CustomTeacherCourseCreation);
```
Will need to be a different name or we'll run into issues when loading the component into the LMS.

File: d2l-custom-teacher-course-creation-admin.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { LocalizeMixin } from '@brightspace-ui/core/mixins/localize-mixin.js';
+
+class CustomTeacherCourseCreation extends LocalizeMixin(LitElement) {
```
Comment (BraydonKains) [inline d2l-custom-teacher-course-creation-admin.js:4]: Nit: this won't break anything, but I think this is the same class name as the other top level component.

Comment (quinnbudan) [reply d2l-custom-teacher-course-creation-admin.js]: Replaced with a more accurate class name

File: d2l-custom-teacher-course-creation-admin.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+	static get properties() {
+		return {
+			prop1: { type: String },
```
[the remark this answers is not in the document]
Comment (BraydonKains) [reply d2l-custom-teacher-course-creation-admin.js]: I'm sure we will asap after getting this in. We're cutting corners to get everything into master before branch cut.

**Pull request merged**